src/SFA.DAS.EmployerAccounts/Queries/GetOrganisationAgreements/GetOrganisationAgreementsQueryHandler.cs

```diff
@@ -49,13 +50,13 @@ public async Task<GetOrganisationAgreementsResponse> Handle(GetOrganisationAgree
 
             var accountLegalEntityId = _accountLegalEntityPublicHashingService.DecodeValue(message.AccountLegalEntityHashedId);
 
-            var accountLegalEntity = await _employerAgreementRepository.GetOrganisationsAgreements(accountLegalEntityId);            
-
+            var accountLegalEntity = await _employerAgreementRepository.GetOrganisationsAgreements(accountLegalEntityId);          
             if (accountLegalEntity == null) return new GetOrganisationAgreementsResponse();
 
             var organisationLookupByIdPossible = await _referenceDataService.IsIdentifiableOrganisationType(accountLegalEntity.LegalEntity.Source);
 
-            var agreements = _mapper.Map<ICollection<EmployerAgreement>, ICollection<EmployerAgreementDto>>(accountLegalEntity.Agreements,
+            var agreements = _mapper.Map<ICollection<EmployerAgreement>, ICollection<EmployerAgreementDto>>
+                (accountLegalEntity.Agreements.Where(x => x.StatusId == EmployerAgreementStatus.Pending || x.StatusId == EmployerAgreementStatus.Signed).ToList(),
```

Comment:
this StatusId check should not be done here.

Reply:
done

Reply:
done